index.html

```diff
@@ -46,45 +46,14 @@
     gtag("config", "UA-149400351-4");
   </script>
 </head>
-
+        <div class="links">
```

Comment:
Why are these links before the body?

Reply:
I'm sorry about that.

I guess it's a mistake, I'd fix it now.